### PR TITLE
Use latest 10up/action-wordpress-plugin-deploy

### PR DESCRIPTION
**Pull request by szepeviktor**
...and make CI configuration much better 😻

### CODE REVIEW

File: .github/workflows/deploy-to-wordpress.yml [hunk 1]
```diff
@@ -5,45 +5,49 @@ on:
...
+
     - name: Install dependencies
-      run: |
-        composer install --no-dev
-    - name: WordPress Plugin Deploy
-      uses: 10up/action-wordpress-plugin-deploy@master
+      run: composer install --no-dev
+
+    - name: Deploy WordPress Plugin
+      uses: 10up/action-wordpress-plugin-deploy@stable
```
Comment (szepeviktor) [inline .github/workflows/deploy-to-wordpress.yml:26]: `@stable` will pick the latest tag

Comment (szepeviktor) [reply .github/workflows/deploy-to-wordpress.yml:26]: Please also see https://github.com/10up/action-wordpress-plugin-deploy/commit/0dc3f76f710b55cd07aa546a77bc103a0f1f2f7a#commitcomment-47435338

**Comment by jasonbahl** [comment]
Closing, as we were doing similar work at the same time.

**Comment by szepeviktor** [comment]
I see.
https://github.com/wp-graphql/wp-graphql/commit/5b1ec270a2ee666ec74ea51e9a542dfa0cbe1e76#diff-383a5c9f8660333ab16cf560cb16d2af0674612167e236ddf410e846be9b0647

**Pull request closed without merging**